members/views.py

```diff
@@ -1,3 +1,82 @@
-from django.shortcuts import render
+import json
+import re
+import bcrypt
+import jwt
 
-# Create your views here.
+from django.views     import View
+from django.http      import JsonResponse
+from django.db.models import Q
+
+from chickenfood.settings import SECRET_KEY
+from members.models       import Member
+
+class AgreementView(View):
```

Comment:
```suggestion
class SignUpView(View):
```
또는 
```suggestion
class MemberView(View):
```
Agreement가 화원가입을 명확히 나타내주지 못 하고 있습니다. view 이름과 url도 수정해 주세요.

Reply:
넵 수정하여 다시 올리겠습니다.